Formula/u/upx.rb

```diff
@@ -29,8 +31,8 @@ def install
     cp bin/"upx", "."
     chmod 0755, "./upx"
 
```

Comment:
Looks like you forgot to remove these lines too. Mind opening a follow-up PR?

Reply:
Done: #188154